### PR TITLE
Try this to add make check stage

### PR DESCRIPTION
**Pull request by bbdocs**

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://12-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://12-bb-docs.surge.sh

File: .travis.yml [hunk 1]
```diff
@@ -5,15 +5,22 @@ os:
   - linux
 node_js:
   - 7
+addons:
+  apt:
+    packages:
+    - hunspell
+    - hunspell-en-us
```
Comment (bbewan) [inline .travis.yml]: We don't need the dictionary package; the `_dicts` folder contains the expanded US dictionary, as well as a BB-specific dictionary.

**Comment by bbdocs** [comment]
See the website here: http://12-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://12-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://12-bb-docs.surge.sh

**Comment by bbdocs** [comment]
phew.  redirects are working on surge site.  trying merge and deploy to preprod.

**Pull request merged**